tests/native/test_manticore.py

```diff
@@ -134,20 +134,3 @@ def test_integration_basic_stdin(self):
         else:
             self.assertTrue(a <= 0x41)
             self.assertTrue(b > 0x41)
-
-
-class ManticoreLogger(unittest.TestCase):
```

Comment:
@feliam I moved this from this file over to [`test_logging.py`](https://github.com/trailofbits/manticore/pull/1620/files#diff-4bc2d75c7a6824cb932ea758bc102631ca7f1d84788a3df755fbb2ae40e07ec2R7) in order to avoid the problem where the log level gets stuck at the max level. I seem to recall you having dealt with something similar, but I can't remember where. Did you come up with a better solution?